interactives/bin-packing/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <!--Let browser know website is optimized for mobile-->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <!-- ### Enter your interactive title in the title block ### -->
+  <title>JPEG Compression</title>
+  <link rel="stylesheet" href="../base-files/css/third-party/normalize.css">
+  <!--Import materialize.css-->
+  <link rel="stylesheet" href="../base-files/css/third-party/materialize.min.css" type="text/css" media="screen,projection" />
+  <link rel="stylesheet" href="./css/bin-packing.css">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <script src="../base-files/js/third-party/modernizr.js"></script>
+  <!-- jQuery (necessary for Materialize) -->
+</head>
+
+<body>
+  <div class="container">
+    <div id="modal1" class="modal">
+      <div class="modal-content">
+        <p>Enter the comma separated item sizes. The sizes should be between 1 and 100.</p>
+            <input type="text" id="sizes" class="validate" minlength="1" pattern="^(([1-9]|[1-8][0-9]|9[0-9]|100)\s*)?(\s*,\s*([1-9]|[1-8][0-9]|9[0-9]|100))*$"/>
```

Comment:
Could you explain what the pattern does?

Reply:
I have removed it. It was an attempt to do input checking, which didn't work and we decided not to prioritise spending time to make it work.